### ISSUE TITLE
Sutro Integrations program

### ISSUE THREAD

**Issue by mgoulet65** [ydogandjiev/hass-sutro#27]
First...THANK YOU for this integration. It makes my pool job so much easier.

Have you seen that Sutro has an Integration program? https://my.mysutro.com/integrations

Would love for this to be official.

**Comment by ydogandjiev** [comment]
I am working on it:
https://twitter.com/ydogandjiev/status/1610124898244120577?s=20&t=rfL-Tq9X_UvEDHzEvlfvqQ

**Comment by mgoulet65** [comment]
Glad they're responsive! Let me know if I can help (not technical but happy to interface with Sutro for you).

**Comment by wonderslug** [comment]
@ydogandjiev Did this go anywhere?

**Comment by ydogandjiev** [comment]
@wonderslug, I'm still discussing this with the Sutro team. They pushed an update to clean up some of the code and have also suggested that I add support for recommendations - #67. Thank you for helping out with this part! They also asked me to improve the security of credential storage; however, I couldn't find a way to make that any more secure unless the Sutro service adds support for OAuth. Their engineers are looking into this now.

The integration did get mentioned in one of their recent videos here - https://youtu.be/Op8PAow5Eu0. I'm still hopeful that we can get the integration officially featured on their site once all the asks have been addressed.

**Comment by wonderslug** [comment]
@ydogandjiev 

Yeah I agree, I don't see a way to change the credentials storage as its the standard way HA does it.  OAuth would be best for them.

Nice the integration got a mention by them. :)
